bedrock/legal/templates/legal/docs-base.html

```diff
@@ -18,9 +18,19 @@
 {% endblock %}
 
 {% block side_nav %}
-<nav class="mzp-c-sidemenu mzp-c-sidemenu-main">
-  <ul>
-    <li><a href="{{ url('legal.index') }}">{{ _('Back to Legal') }}</a></li>
-  </ul>
+<nav class="mzp-c-sidemenu">
+  <section class="mzp-c-sidemenu-summary mzp-js-toggle" aria-controls="sidebar-menu">
+    <h3 class="mzp-c-sidemenu-label">{{ self.page_title() }}</h3>
+    <ul>
+      <li class="mzp-is-current">{{ self.page_title() }}</li>
```

Comment:
The page title is repeated three times in the side menu on most pages, which feels perhaps a little too much?

![image](https://user-images.githubusercontent.com/400117/69715680-8f4bc080-1100-11ea-9c25-52e55fbf3a07.png)

Perhaps just one title, plus the "Back to Legal" link should be enough? I don't think there's all that much to be gained by the pages linking to themselves?